packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -3299,28 +3570,33 @@ namespace Ifpack2 {
           comm_ = *mpi_comm->getRawMpiComm();
 #endif
         }
+        work_[0] = 0; 
+        work_[1] = 0; 
+        work_[2] = -1;
```

Comment:
If `work_` stores `impl_scalar_type`, then these assignments may not compile for the complex arithmetic case.  See note above.

Reply:
work_ is magnitude type. It is anyway fixed as it is more clear and prevent future warnings.

Reply:
Thanks!